rebasehelper/application.py

```diff
@@ -121,6 +121,7 @@ def __del__(self):
 
     @staticmethod
     def setup(cli_conf):
+        MacroHelper.add_macros(cli_conf.rpmmacros)
```

Comment:
No, the option is supposed to populate `SpecFile.predefined_macros`.

Reply:
You are right, it doesn't work. I tested that the macros get parsed correctly but didn't test it on a real package which could benefit from this option. It should be fixed now